README.md

```diff
@@ -12,11 +10,12 @@ The `geofabrics` package includes routines and classes for combining point (i.e.
 
 ![image](https://user-images.githubusercontent.com/22883860/128109715-80738b70-b5d9-480e-811e-1c045c27105f.png)
 
+## Documentation
 ### Wiki
+For detailed instructions on installation, usage, and testing please check out the repo [wiki](https://github.com/rosepearson/GeoFabrics/wiki). For 'install instructions' see [Package Install Instructions](https://github.com/rosepearson/GeoFabrics/wiki/Package-Install-Instructions)
 
-For detailed instructions on installation, usage, and testing please check out the repo [wiki](https://github.com/rosepearson/GeoFabrics/wiki). 
-
-For 'install instructions' see [Package Install Instructions](https://github.com/rosepearson/GeoFabrics/wiki/Package-Install-Instructions)
+### API documentation
+Sphinx combined with GitHub Pages is used to create web-hosted documentation from the embedded docstrings in the source code at [https://rosepearson.github.io/GeoFabrics/](https://rosepearson.github.io/GeoFabrics/)
```

Comment:
This could be less specific, saying that "an auto-generated API documentation is available at https://rosepearson.github.io/GeoFabrics" (I think the url auto-renders as a link, no need for [link](link) markup). I don't think the reader needs to know about sphinx and github-action.

Reply:
Good suggestion, thanks! Done.